packages/core/saved-objects/core-saved-objects-common/src/will_move_to_server.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
```

Comment:
Not sure where else to include this, so adding it here: Can we please name this file something more useful w.r.t what the file contains? Maybe "legacy_saved_objects" or "legacy_saved_objects_common_types"?